src/actions/breakpoints/addBreakpoint.js

```diff
@@ -90,9 +96,15 @@ async function addBreakpointPromise(getState, client, sourceMaps, breakpoint) {
     text,
     originalText
   };
-
+  if (!shiftKey) {
+    newBreakpoint.disabled = false;
+  }
   assertBreakpoint(newBreakpoint);
 
+  if (breakpoint.disabled && shiftKey) {
```

Comment:
Are there cases for line 104 in which the values of `breakpoint.disabled` and `shiftKey` would not be the same? If not, it would be clearer if this were put together with the previous `if(!shiftKey)` to form an if/else statement

Reply:
I didnt know what the behavior should be if someone shift clicks an disabled breakpoint (or why they would) so did it more as a safe guard from malicious users.

Reply:
`addOrToggleDisabledBreakpoint` already handles that case (shift+click on disabled breakpoint) by calling `toggleDisabledBreakpoint`, so by the time you're in `addBreakpointPromise` it's not necessary.

Reply:
ok removed the breakpoint.disabled check.